integration_tests/tests/apply/risks-and-needs/risk-information/complete_additional_concerns_page.cy.ts

```diff
@@ -39,8 +55,24 @@ context('Complete "Additional concerns" page', () => {
     cy.visit('applications/abc123/tasks/risk-information/pages/additional-concerns')
   })
 
-  it('exists', function test() {
-    // Given I am on the Additional concerns page
+  //  Scenario: view additional concerns questions
+  //    Then I see the "additional concerns" page
+  it('presents the additional concerns page', function test() {
     Page.verifyOnPage(AdditionalConcernsPage, this.application)
   })
+
+  //  Scenario: complete page and navigate to next page in risk information task
+  //    When I complete the additional concerns page
+  //    And I continue to the next task / page
+  //    Then I see the "risk management arrangements" page
+  it('navigates to the next page (risk management arrangements)', function test() {
+    AdditionalConcernsPage.visit(this.application)
```

Comment:
strictly speaking I guess this `visit()` call repeats the one in the `beforeEach` block. You could replace it and the line below with `const page = Page.verifyOnPage(AdditionalConcernsPage, this.application)`